docs/content/en/docs/reference/tracing-policy.md

```diff
@@ -73,13 +75,53 @@ metadata:
   name: "fd-install"
 ```
 
-**The specification**
+#### Specification
 
 The `spec` or `specification` can be composed of `kprobes` or `tracepoints` at
 the moment. It describes the arbitrary event in the kernel that will be traced.
+
+##### Kprobes
+
+Kprobes enables you to dynamically break into any kernel routine and collect
+debugging and performance information non-disruptively. kprobes are highly
+tight to your kernel version and might not be portable since the kernel symbols
+depend on your build.
+
+Conveniently, you can list all kernel symbols reading the `/proc/kallsyms`
+file. For example to search for the `write` syscall kernel function, you can
+execute `sudo grep sys_write /proc/kallsyms`, the output should be similar to
+this, minus the architecture specific prefixes.
+
+```
+ffffdeb14ea712e0 T __arm64_sys_writev
+ffffdeb14ea73010 T ksys_write
+ffffdeb14ea73140 T __arm64_sys_write
+ffffdeb14eb5a460 t proc_sys_write
+ffffdeb15092a700 d _eil_addr___arm64_sys_writev
+ffffdeb15092a740 d _eil_addr___arm64_sys_write
+```
+
+You can see that the exact name of the symbol for the write syscall on our
+kernel version is `__arm64_sys_write`. Note that on `x86_64`, the prefix should
+be `__x64_` instead of `__arm64_`.
+
+{{< caution >}}
+Kernel symbols contain an architecture specific prefix when they refer to
+syscall symbols. To write portable tracing policies, i.e. policies that can run
```

Comment:
```suggestion
syscall symbols. It's probably a good idea to write portable tracing policies that can run
```

Reply:
Should this be resolved @mtardy ?

Reply:
I don't agree with the suggestion personally but we can debate

Reply:
both are fine, so let's just merge as it is, and we can improve anyway later! much appreciated @errordeveloper for the feedback

Marked resolved